README.md

```diff
@@ -1,4 +1,5 @@
 # Software Design And Maintenance 
 Documentation for CSCI 630 at CSU Chico
 
-[Roster](roster.md)
\ No newline at end of file
+[Roster](roster.md)
+[Roster](Schedule.txt)
```

Comment:
Let's rename Schedule.txt as Schedule.md so we can use markdown

Also, make the link name "Course Schedule" instead of "Roster"